doctr/models/predictor/pytorch.py

```diff
@@ -25,6 +27,9 @@ class OCRPredictor(nn.Module, _OCRPredictor):
     Args:
         det_predictor: detection module
         reco_predictor: recognition module
+        straighten_pages: if True, evaluates the page general orientation based on the median of each line orientation.
```

Comment:
"evaluates" --> "estimates"
"median of each line orientation" --> "median line orientation"

Reply:
Done